src/WCSimPrimaryGeneratorMessenger.cc

```diff
@@ -44,6 +45,20 @@ WCSimPrimaryGeneratorMessenger::WCSimPrimaryGeneratorMessenger(WCSimPrimaryGener
   fileNameCmdCosmics->SetParameterName("fileName",true);
   fileNameCmdCosmics->SetDefaultValue("data/MuonFlux-HyperK-ThetaPhi.dat");
 
+  hepmc3fileNameCmd= new G4UIcmdWithAString("/mygen/hepmc3file",this);
+  hepmc3fileNameCmd->SetGuidance("Select the file of HepMC3.");
+  hepmc3fileNameCmd->SetGuidance(" Enter the file name of the HepMC3 file");
+  hepmc3fileNameCmd->SetParameterName("fileName",true);
+  hepmc3fileNameCmd->SetDefaultValue("inputhepmc3file");
+
+  hepmc3positionGenCmd = new G4UIcmdWithABool("/mygen/hepmc3positionGen",this);
+  hepmc3positionGenCmd->SetGuidance("Set to generate isotope positions or read from file.");
+  hepmc3positionGenCmd->SetGuidance("true : generate positions, false : read from file");
```

Comment:
```suggestion
  hepmc3positionGenCmd->SetGuidance("true : generate positions randomly, false : read from file");
```

Also... generate positions where? Tank + rock? Entire water volume tank? ID only? ID FV?